.github/workflows/karate.yml

```diff
@@ -0,0 +1,114 @@
+name: Integration Tests
+
+on:
+  schedule:
+    # Run the ci every day at 19:00
+    - cron:  '00 19 * * *'
```

Comment:
how long does it take ? 
By default I'd strongly suggest running it in the early morning, say 4 am, as it's typical for students to work in the evening and 7pm may run "while" they're merging stuff.

Reply:
It takes around 15 minutes.

I can definitely move it to 4 am.

Reply:
It seems to be the default behavior, yes :

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule